packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -109,9 +115,16 @@ function GridItem< Item >( {
 			<titleField.render item={ item } field={ titleField } />
 		) : null;
 
+	const isPicker = Boolean( picking );
+	const isClickable = () => ! isPicker && isItemClickable( item );
```

Comment:
Not a blocker, but I notice that we call this function in a few places in this component, and at other times we pass it down. Would it be worth it to:

* Call it once and store the value for the subsequent checks?
* And/or memoize / useCallback if we need to maintain a reference in any of the cases where we pass it down?

(I didn't notice any issues with this in practice, so this is simply a question)